### ISSUE TITLE
Classification problems

### ISSUE THREAD

**Issue by VincentRichard555** [trevorstephens/gplearn#14]
Hello,

Can gplearn be used to solve classification problems? I have an example running with two classes (0 and 1) and est.score is giving me for accuracy .0017 which is quite low, perhaps due to the targets being classes.

Thanks

EDIT:

Used LogisticRegression instead of Ridge and it works fine

http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html

**Comment by trevorstephens** [comment]
Hi @VincentRichard555 , thanks for the question. Currently there is no support for classification. You can use the `SymbolicTransformer` class to generate correlated variables though. These can then be fed into a standard classifier to get new features. See here for an example: http://gplearn.readthedocs.io/en/latest/examples.html#example-2-symbolic-tranformer
